pkg/client/buildconfigs.go

```diff
@@ -73,7 +74,7 @@ func (c *buildConfigs) Get(name string) (result *buildapi.BuildConfig, err error
 func (c *buildConfigs) WebHookURL(name string, trigger *buildapi.BuildTriggerPolicy) (*url.URL, error) {
 	switch {
 	case trigger.GenericWebHook != nil:
-		return c.r.Get().Namespace(c.ns).Resource("buildConfigs").Name(name).SubResource("webhooks").Suffix(trigger.GenericWebHook.Secret, "generic").URL(), nil
+		return c.r.Get().Namespace(c.ns).Resource("buildConfigs").Name(name).SubResource("webhooks").Suffix(trigger.GenericWebHook.Secret, "generic", strconv.FormatBool(trigger.GenericWebHook.AllowEnv)).URL(), nil
```

Comment:
i have an update ready which implements the proposed look/feel for `oc describe ...` 

i'll push once i've gone through the various regression tests